cmd/celestia/header.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	headerCmd.AddCommand(
+		localHeadCmd,
+		networkHeadCmd,
+		getByHashCmd,
+		getByHeightCmd,
+		syncStateCmd,
+	)
+}
+
+var headerCmd = &cobra.Command{
+	Use:   "header [command]",
+	Short: "Allows interaction with the Header Module via JSON-RPC",
+	Args:  cobra.NoArgs,
+}
+
+var localHeadCmd = &cobra.Command{
+	Use:   "local-head",
+	Short: "Returns the ExtendedHeader from the chain head",
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		client, err := rpcClient(cmd.Context())
+		if err != nil {
+			return err
+		}
+
+		header, err := client.Header.LocalHead(cmd.Context())
+		return printOutput(header, err, nil)
+	},
+}
+
+var networkHeadCmd = &cobra.Command{
+	Use:   "network-head",
+	Short: "Returns the recent ExtendedHeader from the network",
```

Comment:
In general for the description, just use the method docs.

Reply:
done